File: python/assign/modul/fib.py
#!/usr/bin/python
def fib(num):
	v1,v2=0,1
#num=input("enter a num:")
	while v1<=num:
		print v1,
		v1,v2=v2,v1+v2
def series(num):
	for val in range(num):
		print val,

File: C/fib.c
#include<stdio.h>
int fib(int val)
{
           if(val==1) return 0;
           if(val==2) return 1;
           return (fib(val-2)+fib(val-1));
}
int main()
{
        int num,index;
	printf("enter a val:");
	scanf("%d",&num);
         for(index=1;index<=num;index++)
              printf("%d ",fib(index));
}          
